components/eamxx/src/mct_coupling/CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ set (SCREAM_LIBS
      spa
      nudging
      diagnostics
+     ml_correction
```

Comment:
Does this need to only happen if we are on `quartz`?  Could this be the cause of the fails on `mappy`?

Reply:
Hmm, but I need this right? But should it have worked on weaver?

Reply:
Weaver does not build the coupled model, it only does standalone testing.

Reply:
If you want to merge this in master, I would strongly encourage you to make this work on supported E3SM machines (at least on chrysalis). If this only works on an internal LLNL machines, then it should probably remain in a branch...